persistence/ravendb/cluster-configuration.md

```diff
@@ -7,28 +7,16 @@ versions: '[6.0,)'
 
 RavenDB version 4 and higher supports configurations with multiple nodes. Support for configurations with a single leader and multiple watchers was originally introduced under the assumption that it’s safe to run without cluster-wide transactions. However, it was discovered that all databases on RavenDB nodes within a cluster behave as primary nodes. That means that any database in a cluster can accept writes. Committing any data safely requires that the data be successfully persisted across a majority of the nodes within the cluster. Optimistic concurrency control cannot be used in cluster scenarios and the only way to enforce this is by using cluster-wide transactions.
 
-Using cluster-wide transactions correctly is complex. Cluster-wide transactions make use of compare-exchange values in order to guarantee consistent writes across the majority of nodes in the cluster. Each document requires a corresponding compare-exchange value to accomplish that. There’s no mechanism to keep documents and related compare-exchange values in sync; it’s up to the user to manage that. It is a challenge to deal with consistent compare-exchange values when using them to write data.
+Using cluster-wide transactions correctly is complex. Cluster-wide transactions make use of compare-exchange values in order to guarantee consistent writes across the majority of nodes in the cluster. With the current RavenDB api, it’s up to the user to properly implement that across the business data and the NServiceBus data.
 
-Using cluster-wide transactions correctly requires all the following:
```

Comment:
this whole part feels off. It is too much on the explanation side to be at the beginning of the document. Should we just start with a simple "cluster configurations are not supported. The End." And then maybe explain in a dedicated section why, for users that are more interested on the topic?

Reply:
I moved the red box that says it is unsupported to the top, see if that addresses your concern?